pytx3/pytx3/cli/label.py

```diff
@@ -25,8 +25,6 @@ class LabelCommand(command_base.Command):
       $ all-in-one -c te.cfg label false-positive descriptor 12345
```

Comment:
thoughts on updating the help text/doc string here to call out that things marked as 'false-positive' will no longer be returned by default?